File: Data_Structures_Linked_Lists/SinglyLinkedList/src/Node.cs
namespace SinglyLinkedList
{
    public class Node {

        public int Data { get; set; }
        public Node? NextLink { get; set; }

        public Node()
        {
            Data = -1;
            NextLink = null;
        }

        public Node (int data)
        {
            Data = data;
            NextLink = null;
        }

        public string StringifyNode()
        {
            return $@"[{Data} | {NextLink}]";
        }
    }
}

File: data-structures/02-linked-lists/list-implementations/01-singly-linked/java/src/SinglyLinkedList/Node.java
package SinglyLinkedList;

public class Node {
	int data;
	Node nextLink;
	
	public Node() {
		data = -1;
		nextLink = null;
	}
	
	public Node(int data) {
		this.data = data;
		nextLink = null;
	}
	
	public void displayNode() {
		System.out.print("[" + data + " | " + nextLink + "]\n");
	}
}